plugin/black.vim

```diff
@@ -98,15 +98,13 @@ if _initialize_black_env():
 def Black():
   start = time.time()
   fast = bool(int(vim.eval("g:black_fast")))
-  line_length = int(vim.eval("g:black_linelength"))
-  mode = black.FileMode.AUTO_DETECT
-  if bool(int(vim.eval("g:black_skip_string_normalization"))):
-    mode |= black.FileMode.NO_STRING_NORMALIZATION
-  if vim.current.buffer.name.endswith('.pyi'):
-    mode |= black.FileMode.PYI
+  mode = black.FileMode()
+  mode.line_length = int(vim.eval("g:black_linelength"))
```

Comment:
Could you pass these to the constructor instead of setting them later?

Reply:
done